fitbenchmarking/fitbenchmarking
# Options docs, defaults and valid values

#### Description of Work

Fixes #583 

Removes `default_options.ini` and moves the documentation into the read the docs and the default values to the options class. Also, adds `valid` values for certain options.

Docs are [here](https://fitbenchmarking.readthedocs.io/en/583_options_docs/)

#### Testing Instructions

1. Check new docs are complete with respect to the old default options file
2. Check docs for adding new options 
3. Check that there are not references to `default_options.ini` 

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
